### ISSUE TITLE
Can't get it to work on RPI3 b+ using latest raspbian

### ISSUE THREAD

**Issue by salmander** [PropaneDragon/PiHueEntertainment#12]
I've installed Qt on a Pi 3 by using

sudo apt-get install qt5-default
sudo apt-get install qtcreator

changed permission of the executable
`sudo chmod +x ./HueEntertainmentCenter.out`

./HueEntertainmentCenter.out

It can't detect the camera. I have tried both logitech webcam c920 and c270 webcams.
`Right at the beginning I get a warning libEGL DRI2 failed to authenticate`

image here (https://photos.app.goo.gl/5etL464TDuHZwTdV8)

**Comment by PropaneDragon** [comment]
Hi @salmander. You'll also need to install the Qt5 QtMultimedia package. I think it's libqt5multimedia5. Install that and let me know if that fixes it.

**Comment by salmander** [comment]
Hey thanks for swift response. Unfortunately, that didn't help! It is saying `libqt5multimedia5 ` is already the newest version 5.11.3-2.

**Comment by salmander** [comment]
Just noticed that by install `sudo apt-get install qt5-default` and `sudo apt-get install qtcreator`. It has installed `qt 5.11.3`. Could that be the cause?

**Comment by PropaneDragon** [comment]
Yeah, that could be an issue. You'd need to have the same version of Qt as it was built with, or at least the same version of Qt base and Qt Multimedia. I'm guessing you'd have to install qt5multimedia with Qt 5.7. Unfortunately, I'm not entirely sure how to get the 5.7 version of Qt Multimedia, but you could try upgrading both to 5.11 and seeing if that works.

**Comment by salmander** [comment]
Both, qt and qt multimedia are using version 5.11. I am trying to find out how to install 5.7.1 but haven't had any luck. Do u think it is relatively easy to compile this app for 5.11?

See image here: (https://photos.app.goo.gl/us2q6RSZ7s1m8mfH8)

**Comment by PropaneDragon** [comment]
Yeah, it should compile fine on 5.11. I'll have to get around to working on it again at some point and update the Qt version.

**Comment by salmander** [comment]
I have never used Qt before. I am trying to download it on my mac so that I can compile the app for version 5.11. I am kinda lost. It seems qt is not free.

**Comment by PropaneDragon** [comment]
Qt is free for open source projects. If you want to download the binaries for it they're [here](https://www.qt.io/download-qt-installer) or you can get the source code [here](https://wiki.qt.io/Building_Qt_5_from_Git#Getting_the_source_code) and compile it yourself. Because of the size of Qt it can be a pain to compile yourself.

**Comment by salmander** [comment]
Nice, thanks! Let me try again

**Comment by salmander** [comment]
Downloading took ages! I have downloaded the desired version now. I am lost again. I am trying to open the project in Qt Creator, but I can't find any project files in your repo. I think this is outside of my realm now ☹️. I will wait for you to upload the binary with version 5.11.

Thanks for your help
